code/game/objects/structures/false_walls.dm

```diff
@@ -159,8 +162,11 @@
 	walltype = /turf/closed/wall/r_wall
 	mineral = /obj/item/stack/sheet/plasteel
 
+/obj/structure/falsewall/reinforced/examine(mob/user)
+	. += ..()
+	. += span_notice("The outer <b>grille</b> is fully intact.")
```

Comment:
This will put both the regular and the reinforced wall message

Reply:
how do i fix it